recipes/r-pkgfilecache/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set version = '0.1.4' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-pkgfilecache
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/pkgfilecache_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/pkgfilecache/pkgfilecache_{{ version }}.tar.gz
+  sha256: 577f7213f6ea8a81ace4145298e724b84cb6ca15173982a9335f72b23c2826bb
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-downloader
+    - r-rappdirs
+  run:
+    - r-base
+    - r-downloader
+    - r-rappdirs
+
+test:
+  commands:
+    - $R -e "library('pkgfilecache')"           # [not win]
+    - "\"%R%\" -e \"library('pkgfilecache')\""  # [win]
+
+about:
+  home: https://github.com/dfsp-spirit/pkgfilecache
+  license: MIT
+  summary: |
+    Manage optional data for your package. The data can be hosted anywhere, and you have
+    to give a Uniform Resource Locator (URL) for each file. File integrity checks are
+    supported. This is useful for package authors who need to ship more than the 5 Megabyte
+    of data currently allowed by the the Comprehensive R Archive Network (CRAN).
+  license_family: MIT
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
+    - LICENSE
+
+extra:
+  recipe-maintainers:
+    - dipterix
+    - dfsp-spirit
```

Comment:
Additional maintainers must comment that they wish to be included.

Reply:
I want to be included.